src/main/java/com/gb/cornucopia/ClientProxy.java

```diff
@@ -1,32 +0,0 @@
-package com.gb.cornucopia;
```

Comment:
IIRC, the clientproxy/serverproxy shit isn't strictly necessary, but is a useful convention for organizing module init logic. Unless something has changed about the mod-init process, you might consider leaving it. I'm not crazy about the name, so if you can come up with something better, that could work too.